executor/write_test.go

```diff
@@ -2968,6 +2968,541 @@ func (s *testSuite4) TestWriteListColumnsPartitionTable(c *C) {
 	tk.MustQuery("select * from t order by id").Check(testkit.Rows("1 a", "2 b"))
 }
 
+// TestWriteListPartitionTable1 test for write list partition when the partition expression is simple.
+func (s *testSuite4) TestWriteListPartitionTable1(c *C) {
+	tk := testkit.NewTestKitWithInit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("set @@session.tidb_enable_table_partition = 1")
+	tk.MustExec("drop table if exists t")
+	tk.MustExec(`create table t (id int, name varchar(10)) partition by list  (id) (
+    	partition p0 values in (3,5,6,9,17),
+    	partition p1 values in (1,2,10,11,19,20),
+    	partition p2 values in (4,12,13,14,18),
+    	partition p3 values in (7,8,15,16,null)
+	);`)
+
+	// Test add unique index failed.
+	tk.MustExec("insert into t values  (1, 'a'),(1,'b')")
+	_, err := tk.Exec("alter table t add unique index idx (id)")
+	c.Assert(err.Error(), Equals, "[kv:1062]Duplicate entry '1' for key 'idx'")
+	// Test add unique index success.
+	tk.MustExec("delete from t where name='b'")
+	tk.MustExec("alter table t add unique index idx (id)")
+
+	// --------------------------Test insert---------------------------
+	// Test insert 1 partition.
```

Comment:
Replace?

Reply:
? replace what?

Reply:
Replace into

Reply:
Already has `replace test` in line 3060.

Reply:
Sorry, I missed that.